src/site/xdoc/writingchecks.xml

```diff
@@ -670,6 +670,11 @@ public class MethodLimitCheck extends AbstractCheck
         <li>
           Code has to be compilable by <code>javac</code> to get valid violations.
           If it is not, you can get hard to understand parse errors.
+          <ul>
+            <li>
+              This includes code relying on compile-time preprocessors (i.e. Lombok
+            </li>
+          </ul>
         </li>
         <li>
```

Comment:
lets add just new LI, with text like `All code that is generated by compile-time preprocessors (like Lombok) is not visible to Checkstyle`